### DISCUSSION TITLE
Vscode extension -> webview -> default ContextMenu for copy, cut and paste in theia.

### DISCUSSION THREAD

**Discussion by abdelhadyahmed** [eclipse-theia/theia#14482] [Q&A] [answered]
I'd like to know if it is missing feature in Theia or just I can't find the right contribution point for it, also I tried to use 
menu path "webview-context-menu" to add menu items from CommonCommands.paste for example, but it's not working on WebView even I clicked on WebView input and paste it won't work.
so please can help?

**Comment by msujew** [comment, marked as answer]
Hey @abdelhadyahmed,

VS Code webviews have their own context menus (or should at least implement their own), as they're rendered as iframes. We don't have any control over anything happening within that iframe, since it is essentially a separate website, and therefore cannot offer a contribution point for that.

**Reply by abdelhadyahmed** [reply]
okay so if I want to copy paste cut inside vscode WebView I should implement them inside my extension.

**Reply by msujew** [reply]
Yes, exactly.

**Reply by abdelhadyahmed** [reply]
Thank you @msujew.
